kuma/static/styles/components/structure.scss

```diff
@@ -23,18 +23,24 @@ Pre-header
 Header
 ====================================================================== */
 
-@import 'structure/main-header';
-@import 'structure/submenu';
+@import 'structure/page-header/main-header';
+@import 'structure/page-header/submenu';
```

Comment:
Good structural changes :)

Reply:
If you want to preserve the git history for these moved pages you can move them in a separate PR where you don't edit the files. This might be complicated rebase dance so it's not a blocker.